README.md

```diff
@@ -9,8 +9,8 @@ Publishes test results while tests run, rather than waiting for completion like
 ### Pipeline
 
 ```groovy
-realtimeJUnit('target/test-results/*.xml') {
-    sh 'mvn verify'
+realtimeJUnit('**/target/surefire-reports/TEST-*.xml') {
+    sh "mvn -Dmaven.test.failure.ignore=true clean package"
```

Comment:
Do not pass (potentially) interpolated strings to `sh` steps.

```suggestion
    sh 'mvn -Dmaven.test.failure.ignore=true clean package'
```

Reply:
Also probably
```suggestion
    sh 'mvn -Dmaven.test.failure.ignore=true clean verify'
```

Reply:
https://github.com/jenkinsci/junit-realtime-test-reporter-plugin/pull/25